pkgs/development/libraries/wayland/default.nix

```diff
@@ -53,13 +41,13 @@ stdenv.mkDerivation (finalAttrs: {
     sed -i '/os-wrappers-test/d' tests/meson.build
   '';
 
-  outputs = [ "out" "bin" "dev" ] ++ lib.optionals withDocumentation [ "doc" "man" ];
+  outputs = [ "out" "dev" ] ++ lib.optionals withDocumentation [ "doc" "man" ];
   separateDebugInfo = true;
 
   mesonFlags = [
     "-Ddocumentation=${lib.boolToString withDocumentation}"
-    "-Dlibraries=${lib.boolToString withLibraries}"
     "-Dtests=${lib.boolToString withTests}"
+    "-Dscanner=false"
```

Comment:
Idk do we need a comment that there is a separately maintained `wayland-scanner` derivation now which is why this is not to be configurable, or is this self-evident...

Reply:
added a comment.